enthought/edge-examples
# edge-oauth2-app example

- Create an edge-oauth2-app that has a base image and boilerplate Flask Hello World
- Refactor streamlit-app-example to be built on top of edge-oauth2-app

## Testing Instructions

- From `edge-oauth2-app`, do `python bootstrap.py`
- `edm shell -e edge-oauth2-app`
- `python -m ci build`
- The base image can be tested with `python -m ci start`. (It will display a basic Hello World)
- From `streamlit-app-example`, do `python bootstrap.py`
- `edm shell -e edge-streamlit-app`
- `python -m ci build`
- `python -m ci start` (It will display an oauth protected streamlit)
- In an incognito window, copy the streamlit app localhost URL. It should force you to login to the local jupyterhub.
- Try registering and launching the `quay.io/enthought/edge-streamlit-demo:latest` image as a native app in a local edge development deployment
